test/executor/update_test.cpp

```diff
@@ -152,21 +152,25 @@ TEST_F(UpdateTests, UpdatingOld) {
   catalog->CreateDatabase(DEFAULT_DB_NAME, txn);
   LOG_INFO("Bootstrapping completed!");
 
-  optimizer::Optimizer optimizer;
+  std::unique_ptr<optimizer::AbstractOptimizer> optimizer;
+  optimizer.reset(new optimizer::Optimizer);
+
+
   auto& traffic_cop = tcop::TrafficCop::GetInstance();
 
+
   // Create a table first
   LOG_INFO("Creating a table...");
-  auto id_column = catalog::Column(type::TypeId::INTEGER,
-                                   type::Type::GetTypeSize(type::TypeId::INTEGER),
+  auto id_column = catalog::Column(type::INTEGER,
+                                   type::Type::GetTypeSize(type::INTEGER),
```

Comment:
I also don't know, once I rebase it and merge a new type.h and the specified define is in src/include/type/type_id.h